test/system/030-run.bats

```diff
@@ -432,4 +432,19 @@ json-file | f
     is "$output" "$expect" "podman run with --tz=local, matches host"
 }
 
+@test "podman container exists" {
+    rand=$(random_string 30)
+    run_podman 1 container exists myctr
+
+    run_podman create --name myctr $IMAGE /bin/true
+    run_podman container exists myctr
+
+    # Create a container that podman does not know about
+    run buildah from $IMAGE
```

Comment:
We cannot guarantee having buildah in the test environment. I have a test for this (storage containers) in the works, but (sigh) it's in a side branch and I forgot to submit it last week. Basically it runs 'podman build', with a `RUN sleep 15` but with `PODMAN_TIMEOUT=5` so the build is likely to leave behind a buildah container. Unfortunately it also requires other changes to `helpers.bash`. May I submit that next week?

Reply:
What do we need to do to make sure buildah is installed?  It would seem a better solution especially as I add more container/storage features into podman,

Reply:
in CI: go through the build-new-VMs dance. In Fedora/RHEL: add `Requires: buildah` to the `-tests` subpackage.

Reply:
@edsantiago is this OK now?

Reply:
Tests would not pass if it was not.

Reply:
It's OK in CI, but gating tests are failing because of missing buildah. On my list for today.